core/server.go

```diff
@@ -118,6 +118,8 @@ func (params *ServerParams) Start() error {
 
 	// Panic if REST&gRPC are not running
 	if !restIsRunning && !grpcIsRunning {
+		err := common.CreateError(params.Ctx, common.InternalError, "cannot start the server", nil)
```

Comment:
I support the idea of logging something here but I'd restructure it slightly. `common.CreateError` is designed to be returned from a handler:
```go
err := errors.New("REST and gRPC servers cannot both be nil")
log.Error(ctx, err)
panic(err)
```